shell/platform/android/io/flutter/view/TextureRegistry.java

```diff
@@ -124,6 +124,28 @@ interface Callback {
 
     /** This method is not officially part of the public API surface and will be deprecated. */
     void scheduleFrame();
+
+    /**
+     * Returns whether the current rendering path handles crop and rotation metadata.
```

Comment:
I would probably add this is generally 29+ in the details.

Reply:
Done!